### PR TITLE
Add links to note-taking platforms

### PR DESCRIPTION
**Pull request by dstansby**
### Summary

I was reading the book, and thought it would be nice to have a couple of links here.

### List of changes proposed in this PR (pull-request)

- Add links to Google docs and HackMD
- Correct "Hackmd" > "HackMD"

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file: @dstansby

### CODE REVIEW

**Review by paulowoicho — APPROVED** [review]
Thank you @dstansby, your edits look great!

File: book/website/collaboration/remote-collab/remote-collab-meetings.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ We have listed them in chronological order and divided into sections for "before
   - Share guidelines and protocols for chairing and note-taking.
 - Decide if there will be designated speakers with presentations.
   - Share the speaker's guidelines and protocols with them.
-- Establish a shared document that can be updated online for agenda and note-taking. For example, using Google docs or Hackmd.
+- Establish a shared document that can be updated online for agenda and note-taking. For example, using [Google docs](https://docs.google.com) or [HackMD](https://hackmd.io).
```
Comment (erictleung) [inline book/website/collaboration/remote-collab/remote-collab-meetings.md:30]: Capitalization, as presented on their website https://www.google.com/intl/en-GB/docs/about/.

```suggestion
- Establish a shared document that can be updated online for agenda and note-taking. For example, using [Google Docs](https://docs.google.com) or [HackMD](https://hackmd.io).
```

**Comment by erictleung** [comment]
@dstansby great add! Just a small suggestion on capitalization. Otherwise, looks great!

**Review by erictleung — APPROVED** [review]
LGTM! :+1:

**Comment by paulowoicho** [comment]
@all-contributors please add @dstansby for content contributions

**Pull request merged**